crypto/encode_decode/decoder_lib.c

```diff
@@ -647,10 +757,11 @@ static int decoder_process(const OSSL_PARAM params[], void *arg)
 #endif
             || (ERR_GET_LIB(err) == ERR_LIB_X509
                 && ERR_GET_REASON(err) == X509_R_UNSUPPORTED_ALGORITHM))
-            break; /* fatal error; preserve it on the error queue and stop */
+            goto end;
     }
 
  end:
     BIO_free(new_data.bio);
     return ok;
+
```

Comment:
Why the spurious blank line?

Reply:
"oops" is a good reason, innit? 😉 
I'll remove it